README.md

```diff
@@ -60,7 +67,7 @@ indexes into a combined one:
 
 ## C
 ```C
-merger = modulemd_module_index_merger_new()
+merger = modulemd_module_index_merger_new();
```

Comment:
General comment that applies to all of the assignments below:

Unlike Python, you need to declare your C variables ahead of time. In this case, you need `ModulemdModuleIndexMerger *merger =`

Do the same for all of the examples.

Reply:
The current `README` doesn't contain a declaration for C variables, so I thought to follow uniformity. Should I change it for them too then? I thought its implied.

Reply:
Hmm, you're correct. The existing README probably should be updated. You don't need to do so as part of this PR.

Reply:
Sorry, I was unclear with this message. You don't need to fix the existing README to have the variable declarations, but I would like you to add them for the new tutorial you are writing. We can fix the lack of them from the existing section in a future PR.